lib/ramble/ramble/application.py

```diff
@@ -1866,11 +1851,11 @@ def is_numeric(value):
                 continue
 
             # When strict success is off for repeats (loose success), skip failed exps
-            if exp_inst.results["RAMBLE_STATUS"] == "FAILED":
+            if exp_inst.result.status == "FAILED":
```

Comment:
(This is unrelated to this PR directly) Can we change "FAILED" to `experiment_status.FAILED` ?

Reply:
Good call, done